cmd/servicelog/common.go

```
@@ -73,59 +74,73 @@ func applyFilters(ocmClient *sdk.Connection, filters []string) ([]*v1.Cluster, e
 func sendRequest(request *sdk.Request) *sdk.Response {
```

You can return a error in this func